<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1561</width>
    <height>999</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>700</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
        <widget class="QComboBox" name="mode_options">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Uniform Range</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Musical Instruments</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Animal Sounds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ECG Abnormalities</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Smoothing Window</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Rectagular</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hanning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hamming</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gaussian</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Original Audio</string>
         </property>
        </widget>
        <widget class="QFrame" name="original_frame">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>281</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#cccccc;
border-radius:15px</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="play_pause_audio_button">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>31</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:black;
</string>
          </property>
          <property name="text">
           <string>►</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QSlider" name="audio_progress">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
                            border: 1px solid #999999;
                            height: 2ps;     
                        }

                       QSlider::handle:horizontal {
                             background: #eb583b;
                             width: 10px;
                            margin: -5px -1px ;
                            border-radius: 5px;
                            border: 1px solid #2a2a2a;} 
QSlider::add-page:horizontal{
        background: qlineargradient(x1:0, y1:0, x2:0, stop:0 #B1B1B1, stop:1 #c4c4c4;)
}
QSlider::sub-page:horizontal{
        background: #eb583b;
}
QSlider::handle:horizontal:hover {
                             background: #007ad9;
                             border-color: #007ad9;

} </string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>310</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Modified Audio</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>281</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#cccccc;
border-radius:15px</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QSlider" name="audio_progress_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
                            border: 1px solid #999999;
                            height: 2ps;     
                        }

                       QSlider::handle:horizontal {
                             background: #eb583b;
                             width: 10px;
                            margin: -5px -1px ;
                            border-radius: 5px;
                            border: 1px solid #2a2a2a;} 
QSlider::add-page:horizontal{
        background: qlineargradient(x1:0, y1:0, x2:0, stop:0 #B1B1B1, stop:1 #c4c4c4;)
}
QSlider::sub-page:horizontal{
        background: #eb583b;
}
QSlider::handle:horizontal:hover {
                             background: #007ad9;
                             border-color: #007ad9;

} </string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="play_pause_audio_button_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>31</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:black;
</string>
          </property>
          <property name="text">
           <string>►</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Spectrogram</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_sliders">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>400</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QSlider" name="uni_slider1">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider3">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
border: 1px solid;
height: 10px;
margin: 0px;
}
QSlider::handle:horizontal {
background-color: black;
border: 1px solid;
height: 40px;
width: 40px;
margin: -15px 0px;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider6">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider5">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider4">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider9">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider8">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider7">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="uni_slider10">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="music_slider1">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>20</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="music_slider2">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>10</y>
           <width>20</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="music_slider3">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>10</y>
           <width>20</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="music_slider4">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>10</y>
           <width>20</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="animal_slider1">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="animal_slider2">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="animal_slider3">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="animal_slider4">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="ecg_slider">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>10</y>
           <width>21</width>
           <height>211</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq1">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq3">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq4">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq5">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq6">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq7">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq8">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq9">
         <property name="geometry">
          <rect>
           <x>670</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="uni_freq10">
         <property name="geometry">
          <rect>
           <x>750</x>
           <y>230</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: yellow</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>991</width>
          <height>271</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGraphicsView" name="graphicsView_original">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>981</width>
           <height>261</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>981</width>
           <height>261</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>991</width>
          <height>271</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>600</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGraphicsView" name="graphicsView_modified">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>981</width>
           <height>271</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>981</width>
           <height>271</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1561</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpload"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpload">
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
